cupyx/scipy/signal/windows/_windows.py

```diff
@@ -2036,6 +2121,64 @@ def taylor(M, nbar=4, sll=30, norm=True, sym=True):
     return _truncate(w, needs_trunc)
 
 
+def lanczos(M, sym=True):
```

Comment:
To get aligned with SciPy.
```suggestion
def lanczos(M, *, sym=True):
```